### PR TITLE
CONJ-4525 Remove docs from conjur repo

### PR DESCRIPTION
**Pull request by mizziness**
Closes CONJ-4525

#### What does this pull request do?

This PR removes the old docs (website) folder and dependencies out from this repo.  They are being relocated to `cyberark/conjur-org`

#### Where should the reviewer start?

Start by checking all of the files required for normal building and testing of the project - make sure that the hosted Conjur server pieces remain functional.

#### How should this be manually tested?

You can get the last successful deb here:  https://jenkins.conjur.net/job/cyberark--conjur/job/WIP-removing-docs/lastSuccessfulBuild/artifact/

You can also clone this branch locally and build in your environment as you would test any other conjur branch or PR.

#### Link to build in Jenkins (if appropriate)

https://jenkins.conjur.net/job/cyberark--conjur/job/WIP-removing-docs/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,23 +186,6 @@ root@9feae5e5e001:/src/conjur/cucumber/api# cucumber
 root@9feae5e5e001:/src/conjur# cucumber -r cucumber/api/features/support -r cucumber/api/features/step_definitions cucumber/api/features/resource_list.feature
 ```
 
-## Documentation site
```
Comment (jvanderhoof) [inline README.md]: What do you think about linking to the new `conjur-org` repo to let people know that docs have moved?

File: docs/README.md [hunk 2]
```diff
@@ -1,66 +0,0 @@
...
-site and will report broken links as errors. It's automatically run as
-part of our continuous integration pipeline, but it's a good idea to
-run it yourself whenever you add or remove links.
-
-[dependencies]: ../README.md#Development_Dependencies "Development Dependencies"
-[proofer]: https://github.com/gjtorikian/html-proofer "HTML Proofer"
-
----
-
-### API Documentation
```
Comment (jvanderhoof) [inline docs/README.md]: Don't we want to keep the API docs in this project?

File: README.md (hunk 1 above)
Comment (mizziness) [reply README.md]: The task breakdown has me adding a README into the /docs folder (which is what I am currently working on) or would you rather it be here?

File: docs/README.md (hunk 2 above)
Comment (mizziness) [reply docs/README.md]: This file is inside /docs/ so I am moving that part of the README into a new one inside /apidocs/ instead.

File: README.md (hunk 1 above)
Comment (jvanderhoof) [reply README.md]: Nope.  Let's add a readme to `docs/` as stated in the story.

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Conjur Docs
+
+The Conjur documentation site (which runs at [conjur.org](http://www.conjur.org))
```
Comment (jvanderhoof) [inline docs/README.md]: Technically the docs don't "run".  How about something like 
```
The Conjur documentation site (available at [conjur.org](http://www.conjur.org))"?
```

File: apidocs/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+If you're working on the API docs locally, you can get a live-updating
+dev server:
+
+```sh-session
+$ cd $(git rev-parse --show-cdup) # start in the project root
+$ docker-compose run --rm --service-ports apidocs -w
+```
+
+Then `open localhost:3000` to see the API docs in your browser.
```
Comment (jvanderhoof) [inline apidocs/README.md]: It feels like this line doesn't make a lot of sense here as the result of the command `docker-compose run --rm --service-ports apidocs -w` runs until the user exits.  How about changing it to something like:
```
Then, navigate to http://localhost:3000 to see the API docs in your browser.
```

Comment (mizziness) [reply apidocs/README.md]: Good catch!  That was a holdover from the old docs.  Fixed!

**Pull request merged**